go.mod

```
@@ -1,6 +1,8 @@
 module github.com/influxdata/influxdb
 
-go 1.21
+go 1.21.0
+
+toolchain go1.21.10
```

this can be simplify:

```suggestion
go 1.21
```